lib/stdlogger.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const EventEmitter = require('events');
+const Hoek = require('@hapi/hoek');
+
+const internals = {};
+
+class Stdlogger extends EventEmitter {
+    constructor(stdout, stderr) {
+
+        super();
+        this.stdout = stdout;
+        this.stderr = stderr;
+    }
+}
+
+[
+    'emergency',
+    'alert',
+    'critical',
+    'error'
+].forEach((level) => {
+
+    Stdlogger.prototype[level] = function (msg, data, additionalFields) {
+
+        const stringified = internals.stringify(level, msg, data, additionalFields);
+        this.stderr.write(`${stringified}\n`);
+    };
+});
+
+[
+    'warning',
+    'notice',
+    'info',
+    'debug',
+    'log'
+].forEach((level) => {
+
+    Stdlogger.prototype[level] = function (msg, data, additionalFields) {
+
+        const stringified = internals.stringify(level, msg, data, additionalFields);
+        this.stdout.write(`${stringified}\n`);
+    };
+});
+
+internals.stringify = function (level, msg, data, additionalFields = {}) {
```

Comment:
I am interested how errors and some common hapi objects will serialize here.

Reply:
That's probably something to add later on or do you see this as part of this first version?

Reply:
We can add those later, the response has a well structured object:

```
{
  "level": "info",
  "time": 1613956324036,
  "msg": "request completed",
  "name": "example",
  "pid": 12340,
  "data": {
    "req": {
      "id": "1613956324029:someID:10000",
      "method": "get",
      "path": "/",
      "headers": {
        "host": "localhost:8080",
        "user-agent": "curl/7.71.1",
        "accept": "*/*"
      },
      "remoteAddress": "127.0.0.1",
      "remotePort": 52525,
      "query": {}
    },
    "res": {
      "statusCode": 200,
      "headers": {
        "content-type": "text/html; charset=utf-8",
        "cache-control": "no-cache",
        "content-length": 7,
        "accept-ranges": "bytes"
      }
    },
    "responseTime": 7
  }
}
```

We can do something similar for the request and other events later